.github/workflows/check-media-attachments.yml

```
@@ -0,0 +1,161 @@
+jobs:
+  check-media-attachments:
+    runs-on: ubuntu-latest
+    name: Check media attachments in HTML files
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check for media attachments in HTML files
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)